package-res/resources/web/pentaho/visual/role/mapping.js

```diff
@@ -142,16 +134,25 @@ define([
       },
 
       /**
+       * Gets a value that indicates if the mapping has any attributes.
+       *
+       * @type {boolean}
+       */
+      get isMapped() {
+        return this.attributes.count > 0;
+      },
+
+      /**
        * Gets the level of measurement in which the visual role will effectively be operating on,
```

Comment:
Suggest:

* Gets the level of measurement on which the visual role will effectively be operating,